backend/metasfresh-webui-api/src/main/java/de/metas/ui/web/quickinput/ddorderline/DDOrderLineQuickInputProcessor.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * #%L
+ * metasfresh-webui-api
+ * %%
+ * Copyright (C) 2020 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.ui.web.quickinput.ddorderline;
+
+import com.google.common.collect.ImmutableSet;
+import de.metas.adempiere.gui.search.IHUPackingAware;
+import de.metas.adempiere.gui.search.IHUPackingAwareBL;
+import de.metas.adempiere.gui.search.impl.DDOrderLineHUPackingAware;
+import de.metas.handlingunits.model.I_DD_OrderLine;
+import de.metas.product.ProductId;
+import de.metas.ui.web.quickinput.IQuickInputProcessor;
+import de.metas.ui.web.quickinput.QuickInput;
+import de.metas.ui.web.window.datatypes.DocumentId;
+import de.metas.util.Services;
+import lombok.NonNull;
+import org.adempiere.exceptions.AdempiereException;
+import org.adempiere.model.InterfaceWrapperHelper;
+import org.adempiere.warehouse.LocatorId;
+import org.adempiere.warehouse.WarehouseId;
+import org.adempiere.warehouse.api.IWarehouseBL;
+import org.eevolution.model.I_DD_Order;
+
+import java.math.BigDecimal;
+import java.util.Set;
+
+public class DDOrderLineQuickInputProcessor implements IQuickInputProcessor
+{
+	private final IWarehouseBL warehouseBL = Services.get(IWarehouseBL.class);
+	private final IHUPackingAwareBL huPackingAwareBL = Services.get(IHUPackingAwareBL .class);
+
+	@Override
+	public Set<DocumentId> process(final QuickInput quickInput)
```

Comment:
pls consider extracting the logic from this method and introduce 

int IDDOrderBL.addLine(DDOrderLineCreateRequest request)
which basically creates the dd order line and returns the DD_OrderLine_ID.

Ideally, DDOrderLineQuickInputProcessor shall contain glue code.... as much as possbile.